app/views/pages/enrolment/i_belong.html.erb

```diff
@@ -17,10 +17,17 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full-width video-with-text">
         <section class='ncce-section govuk-!-margin-top-0 govuk-!-margin-bottom-0'>
-          <p class="govuk-body">Computer science is the fastest-growing STEM subject, and yet, despite its popularity, girls are consistently outnumbered by boys. In 2023, only one in five GCSE Computer Science and 15% of all A level Computer Science entries were from female students in England.</p>
-          <p class="govuk-body">Our evidence-informed programme aims to help teachers and leaders understand the barriers to girls’ participation in computer science and make a plan to overcome them. We offer curated resources, training and implementation support to empower you with the tools to support more girls into computer science qualifications and careers.</p>
+          <p class="govuk-body">Computer science is the fastest-growing STEM subject, and yet, despite its popularity, girls are consistently outnumbered by boys. The gender gap in computing can be traced back to gender stereotypes developed in the primary years. To improve the diversity both in and beyond the classroom, we need to show girls from a young age that they belong in computer science.</p>
+          <p class="govuk-body">Our <%= link_to "evidence-informed", gender_insights_in_computing_url %> programme aims to help teachers and leaders understand the barriers to girls’ participation in computer science and make a plan to overcome them. We offer curated resources, training and implementation support to empower you with the tools to help develop positive attitudes towards computing among female students, resulting in more girls choosing computer science qualifications and careers.</p>
         </section>
-        <%= video_tag("https://static.teachcomputing.org/How_important_is_the_I_Belong_programme.mp4", controls: true) %>
+        <div>
+          <%= video_tag("https://static.teachcomputing.org/How_important_is_the_I_Belong_programme.mp4", controls: true) %>
```

Comment:
I wonder if this video is on the youtube account? Might be one to check with Vicky, as then we can move this from static, and use your youtube component (be easier for future strapifcation)

Reply:
@vickymg I had a check on the TC YouTube channel and I don't see this video - I know some aren't published. Would you know if this video is on YouTube? If so, we can move this from static.

Reply:
Thanks @vickymg - as we don't know how long it will take comms to get the video on YouTube, we will move on with this. When they confirm the video is up on YouTube a new ticket can be created to update.

Reply:
Ticket is here with YouTube link, for when needed: https://github.com/NCCE/teachcomputing.org-issues/issues/2840